libs/data-mapper-v2/src/components/functions/Panel.tsx

```diff
@@ -0,0 +1,75 @@
+      fill="#0F6CBD"
+    />
+  </svg>
+);
+
+export const Panel = (_props: PanelProps) => {
+  const [expandFunctionsPanel, setExpandFunctionsPanel] = useState(false);
+  const styles = useStyles();
+  const intl = useIntl();
+
+  const stringResources = useMemo(
+    () => ({
+      FUNCTIONS: intl.formatMessage({
+        defaultMessage: 'Functions',
+        id: 'zsc+jp',
+        description: 'Functions',
+      }),
+      SEARCH_FUNCTIONS: intl.formatMessage({
+        defaultMessage: 'Search Functions',
+        id: '2xQWRt',
+        description: 'Search Functions',
+      }),
+    }),
+    [intl]
+  );
+
+  return (
+    <InlineDrawer
+      className={expandFunctionsPanel ? styles.expandedDataMapperFunctionPanel : styles.collapsedDataMapperFunctionPanel}
+      open={true}
+      size="small"
+    >
+      {expandFunctionsPanel ? (
+        <div className={styles.expandedDrawerBodyWrapper}>
+          <div className={styles.drawerHeaderWrapper}>
+            <h2 className={styles.drawerHeader}>{stringResources.FUNCTIONS}</h2>
+            <ChevronDoubleLeftRegular
+              fontSize={15}
+              className={styles.drawerHeaderIcon}
```

Comment:
Can we put the font size in the `styles.drwawerHeaderIcon`?

Reply:
Ahh this fontSize is a property on the icon itself